.gitignore

```diff
@@ -0,0 +1,294 @@
+## Ignore Visual Studio temporary files, build results, and
+## files generated by popular Visual Studio add-ons.
```

Comment:
@SebastianSchuetze nice to have: YAML examples for the people using yaml build definitions

Reply:
I will check and add some examples. How detailed do you want to get?
Currently this one supports inline XML and linked XML files. If you could name some use cases, that could be interesting, then I can add some YAML examples. But these examples would not be included in the extension of course but would function as documentation on how to use this task with YAML.

Reply:
I think one YAML abstract per task with all the parameters of the task, example values and a comment to state what the parameter does will be enough.
IMHO something like that for each task will be more than enough https://docs.microsoft.com/en-us/azure/devops/pipelines/tasks/package/npm?view=azure-devops#yaml-snippet

Reply:
ok. I will create a yaml example. Maybe I make it similar to the page your reference as a markdown help with help docs.